.vscode/extensions.json

```diff
@@ -9,7 +9,8 @@
         "editorconfig.editorconfig",
         "aaron-bond.better-comments",
         "davidanson.vscode-markdownlint",
-        "jpoissonnier.vscode-styled-components"
+        "jpoissonnier.vscode-styled-components",
+        "christian-kohler.path-intellisense"
```

Comment:
How about adding `{ "typescript.suggest.paths": false }` to `.vscode/settings.json`, despite it already works well.

Reply:
OK so `typescript.suggest.paths` only suggests for `import` and `require` calls. In case of `new URL("./relative path", import.meta.url)` it does not suggests anything

Reply:
https://github.com/microsoft/TypeScript/issues/43073 I add a feature request to TypeScript